### ISSUE TITLE
Proposal: Example for Independent Latitude, Longitude, Non-spatiotemporal Variable, and Time Axes

### ISSUE THREAD

**Issue by GeyerB** [cf-convention/cf-conventions#249]
# Title
 Example for Independent Latitude, Longitude, Non-spatiotemporal Variable, and Time Axes
# Moderator
@GeyerB

# Moderator Status Review [last updated: 20/03/02]
no update yet

# Requirement Summary
Analogue to chapter `5.1. Independent Latitude, Longitude, Vertical, and Time Axes`, where the use of spatiotemporal dimensions is explained we should explain how to structure data with `Independent Latitude, Longitude, Non-spatiotemporal Variable, and Time Axes`. The new class of standard_names starting with `probability_distribution_of` belongs to that group.

We could have a new subchapter of chapter 5 or an annex with additional examples for all sorts of such examples coming in future.
# Technical Proposal Summary
**Independent Latitude, Longitude, Non-spatiotemporal  Variable, and Time Axes**
For probability distributions of variables, dimensions are used to locate data values in time and space and in the range of the variable itself.

**Example: Independent coordinate variables with non-spatiotemporal  variables**
```
dimensions:
  time = UNLIMITED; // (5 currently)
  nbin = 36;
  nv = 2;
variables:
  double time(time);
    time: standard_name = "time";
    time: long_name = "time" ;
    time: units = "days since 1970-01-01 00:00:00" ;
    time: bounds = "time_bnds";
  double time_bnds(time,nv);
  float class(nbin);
    class: standard_name = "wind_from_direction"
    class: long_name = "wind sector";
    class: units = "degree";
    class: bounds = "class_bnds";
  float class_bnds(nbin,nv);
  float WDIR_freq(time,lat,lon,class);
    WDIR_freq: standard_name = "probability_distribution_of_wind_from_direction_over_time";
    WDIR_freq: units = "1";
    WDIR_freq: coordinates = "lon lat class"
    WDIR_freq: cell_methods = "time: sum (interval: 40s)"
```

# Benefits
Users of the new category of standard_names starting with probability_distribution get a roadmap how to construct such data files.
# Status Quo
No example available.
# Detailed Proposal
see above

**Comment by JonathanGregory** [comment]
I agree that it would be a good idea to add such an example. Thanks.

**Comment by JonathanGregory** [comment]
Dear @GeyerB

You raised this issue in 2020, proposing to add an example of the use of a non-spatiotemporal axis. I think it's a good idea, as I did then. Are you still willing to carry this forward?

Best wishes

Jonathan
